community/rpm.sh

```diff
@@ -78,10 +78,39 @@ detect_os ()
   echo "Detected operating system as ${os}/${dist}."
 }
 
+add_pgdg ()
```

Comment:
And checking this in Red Hat is:

```sh
yum list -q postgresql95-server 2>/dev/null
```

Again, can check for empty string or use exit code.